poly-commitment/src/commitment.rs

```diff
@@ -232,7 +230,7 @@ impl<'a, 'b, C: AffineCurve> Add<&'a PolyComm<C>> for &'b PolyComm<C> {
     }
 }
 
-impl<'a, 'b, C: AffineCurve> Sub<&'a PolyComm<C>> for &'b PolyComm<C> {
+impl<'a, 'b, C: AffineRepr + Sub<Output = C::Group>> Sub<&'a PolyComm<C>> for &'b PolyComm<C> {
```

Comment:
LGTM, even though I am surprised you do have to add this additional trait. Can we not get rid of it?

Reply:
It is better to leave it like than going into projective coordinates and come back.

Reply:
It seems that before, there were simply doing in affine coordinates: https://github.com/arkworks-rs/algebra/blob/v0.3.0/ec/src/models/short_weierstrass_jacobian.rs#L246.